INSTALL.md

```diff
@@ -235,19 +235,3 @@ Building on 10.10 with 10.9 compatibility:
 ```bash
 cmake .. -DCMAKE_OSX_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk  -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9
 ```
-
```

Comment:
Why did you remove parts of the 0.9.3 instructions? Just leave them in place please until the new version is released